### DISCUSSION TITLE
When we navigate, how to keep a radio button selected (by default)

### DISCUSSION THREAD

**Discussion by yashhaste5** [Budibase/budibase#9173] [Help]
I have two screens (screen1 and screen2). And on screen2 have added a radio button, which shows only the selected value from screen1.
When I navigate from screen1 to screen2 using a icon, I am able to show the selected value in radio button (on screen1).
But it only shows the value, have click on it to select it.

What I want to do is, when I navigate from screen1 to screen2 the radio button should be default selected, is it possible on budibase. 
If yes, how can I achieve it.

**Comment by melohagan** [comment]
Hey @yashhaste5 

Could you show some screenshots please to help me visualize the problem?

I'm not sure what you meant by this:
> But it only shows the value, have click on it to select it.
